katello/rubygem-hammer_cli_foreman_virt_who_configure/rubygem-hammer_cli_foreman_virt_who_configure.spec

```diff
@@ -0,0 +1,79 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+
+%global gem_name hammer_cli_foreman_virt_who_configure
+%global confdir hammer
+
+Summary: Hammer CLI commands for configuring Virt Who for Katello
+Name:    %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.0.3
+Release: 1%{?dist}
+Group:   Development/Languages
+License: GPLv3
+URL:     https://github.com/theforeman/hammer-cli-foreman-virt-who-configure
+Source0: https://rubygems.org/downloads/%{gem_name}-%{version}.gem
+
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+%if 0%{?scl:1}
+Obsoletes: rubygem-%{gem_name} < 1.0.1-7
+%endif
+
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(hammer_cli) >= 0.5.0
+Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) >= 0.5.0
+
+BuildRequires: %{?scl_prefix_ruby}ruby(rubygems)
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+
+%description
+Hammer CLI commands for configuring Virt Who for Katello
+
+%package doc
+Summary:   Documentation for %{pkg_name}
+Group:     Documentation
+Requires:  %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{pkg_name}
+
+%prep
+%setup -n %{pkg_name}-%{version} -q -c -T
+%{?scl:scl enable %{scl} - <<EOF}
+%gem_install -n %{SOURCE0}
+%{?scl:EOF}
+
+%install
+mkdir -p %{buildroot}%{_root_sysconfdir}/%{confdir}/cli.modules.d
+install -m 755 .%{gem_instdir}/config/foreman_virt_who_configure.yml %{buildroot}%{_root_sysconfdir}/%{confdir}/cli.modules.d/foreman_virt_who_configure.yml
```

Comment:
Also executable

Reply:
From what I can tell, this is true in all hammer packages. I'd prefer not to one off change a few and make this change across the board.